unison-src/builtin-tests/testlib.u

```diff
@@ -1,3 +1,4 @@
+use builtin.Universal ==
```

Comment:
Use `===` for universal / structural equality, it's in base: https://share.unison-lang.org/@unison/p/code/latest/namespaces/public/base/latest/;/terms/Universal/%3D%3D%3D